.cirrus.yml

```diff
@@ -42,8 +42,7 @@ linux_arm_task:
         - lscpu || true
         - (cat /proc/meminfo | grep MemTotal) || true
         - FISH_TEST_MAX_CONCURRENCY=6 build_tools/check.sh
-    # CI task disabled during RIIR transition
-    only_if: false && $CIRRUS_REPO_OWNER == 'fish-shell'
+    only_if: $CIRRUS_REPO_OWNER == 'fish-shell'
```

Comment:
I think we want something like `only_if: repo owner is fish-shell` also for
`.github/workflows/docker_builds.yml`,
else these might run (and typically fail) whenever someone pushes to their github fork's `master` branch
(there's usually not too much need for anyone to do that, but people do and it does make some things easier for the lazy
(by default github workflows only run from master, unless `gh workflow run --ref` is used etc)